### PR TITLE
Updated Readme file and added pull request template(#2q9nxqt)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<!-- If you made any changes in the UI layer, please provide before/after screenshots -->
+
+
+**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change
+
+
+### Contribution and Currently Important Rules Acceptance
+<!-- Please get familiar with following info -->
+
+- [ ] I read and followed [contribution rules](https://github.com/DivanteLtd/vsf-capybara/blob/master/CONTRIBUTING.md)
```
Comment (ymaheshwari1) [inline .github/PULL_REQUEST_TEMPLATE.md:20]: Update this link to admin-helper repo.

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+name: Run build tests
+
+'on':  
+  push:
+  pull_request:
+    branches:
+      - master
```
Comment (ymaheshwari1) [inline .github/workflows/pull-request.yml:7]:
```suggestion
      - main
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,93 @@
...
-shopify-admin-helper
+# Prerequisite
+Ionic CLI - If you don't have the ionic CLI installed refer [official documentation](https://ionicframework.com/docs/intro/cli) for the installation instructions.
+
+# Build Notes (Users)
+
+1. Download the app from [release](https://github.com/hotwax/shopify-admin-helper/releases) page and extract it.
+2. Go to the app directory.
+3. Run following command to download dependencies 
+`npm i
```
Comment (ymaheshwari1) [inline README.md:9]:
```suggestion
`npm i`
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,93 @@
...
+ionic capacitor open ios
+
+# Contribution Guideline
+
+1. Fork the repository and clone it locally from the `main` branch. Before starting your work make sure it's up to date with current `main` branch.
+2. Pick an issue from [here](https://github.com/hotwax/shopify-admin-helper/issues). Write in the issue comment that you want to pick it, if you can't assign yourself. **Please stay assigned to one issue at a time to not block others**.
+3. Create a branch for your edits. Use the following branch naming conventions: **shopify-admin-helper/issue-number**.
+4. Please add issue number to your commit message.
+5. Propose a Pull Request to `main` branch containing issue number and issue title.
+6. Use [Pull Request template](https://github.com/hotwax/preorder/blob/main/.github/PULL_REQUEST_TEMPLATE.md) (it's automatically added to each PR) and fill as much fields as possible to describe your solution.
```
Comment (ymaheshwari1) [inline README.md:70]: Update the link.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,93 @@
...
+
+While writing issues, please be as specific as possible. All requests regarding support with implementation or application setup should be sent to.
+# UI / UX Resources
+You may find some useful resources for improving the UI / UX of the app <a href="https://www.figma.com/file/bVPRRw282CqGKMdbz7dciH/HC-Ionic-design-system?node-id=15226%3A38906" target="_blank">here</a>.
+
+# Join the community on Discord
+If you have any questions or ideas feel free to join our <a href="https://discord.gg/SwpJnpdyg3" target="_blank">Discord channel</a>
+# The license
+
+Pre-order app is completely free and released under the Apache v2.0 License. Check <a href="https://github.com/hotwax/shopify-admin-helper/blob/main/LICENSE" target="_blank">LICENSE</a> for more details.
```
Comment (ymaheshwari1) [inline README.md:91]:
```suggestion
Shopify Admin Helper app is completely free and released under the Apache v2.0 License. Check <a href="https://github.com/hotwax/shopify-admin-helper/blob/main/LICENSE" target="_blank">LICENSE</a> for more details.
```

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (azkyakhan) [reply .github/PULL_REQUEST_TEMPLATE.md]: Updated Sir

File: .github/workflows/pull-request.yml (hunk 2 above)
Comment (azkyakhan) [reply .github/workflows/pull-request.yml]: Replaced master with main

File: README.md (hunk 4 above)
Comment (azkyakhan) [reply README.md]: Updated it Sir

File: README.md (hunk 5 above)
Comment (azkyakhan) [reply README.md]: Improved app name

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**